content/04_particles.html

```diff
@@ -574,9 +574,9 @@ <h2>4.3 The Array</h2>
       doesn’t cause the program to crash (as it does with adding), the problem
       is almost more insidious in that it leaves no evidence. To discover the
       flaw I must first establish an important fact. When an object is
-      removed from the array with<code>splice()</code>, all elements are shifted one spot
+      removed from the array with <code>splice()</code>, all elements are shifted one spot
       to the left. Note the diagram below where particle C (index 2) is removed.
-      particles A and B keep the same index, while particles D and E shift from
+      Particles A and B keep the same index, while particles D and E shift from
```

Comment:
I think I will use the lower case p when referring to particles generally and capital `Particle` when talking about the Particle class.

```suggestion
      particles A and B keep the same index, while particles D and E shift from
```